### PR TITLE
News scrapper

### LINKED ISSUE #495: News Scrapper (India Today)
# Aim

- The Script will scrap the news details from times of india as per user and store it in a form of csv file.

# Details

- it will be working on modules like selenium, csv etc
- the scrapping will be based on the user.
- the scraped data will be in proper format and would be stored in csv file.

### PR DESCRIPTION
**Pull request by paulamib123**
# Description

This Script is used to get top 10 headlines from India Today based on the category entered by the user and stores it in a CSV file.  

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: News_Scrapper/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## Output
+
+![script.png](https://i.postimg.cc/CKRH9rBD/script.png)
+
+
+![excel-csv.png](https://i.postimg.cc/ZKmH0L2t/excel-csv.png)
+
+## Author
+
+#### By [Paulami Bhattacharya](https://github.com/paulamib123)
```
Comment (kaustubhgupta) [inline News_Scrapper/README.md:40]: Level two (##)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
The script works perfect, just a minor readme update

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

File: News_Scrapper/README.md (hunk 1 above)
Comment (paulamib123) [reply News_Scrapper/README.md]: I made the changes, Please have a look.

**Review by santushtisharma10 — APPROVED** [review]
@paulamib123 💯

**Review by kaustubhgupta — APPROVED** [review]
Perfect, did the same I was expecting 😄👍

**Review by avinashkranjan — APPROVED** [review]
LGTM.....🚀🚀🚀

**Pull request merged**